### PR TITLE
mgr/dashboard: Add description to menu items on mobile navigation

### PR DESCRIPTION
**Pull request by Exotelis**
Adds the title of the icons as description right next to it if the resolution
of the browser is below 768px.
In addtition, I've added some sccs variables that should be used when working with media queries. Those variables define the bootstrap breakpoints.

- [x] References tracker ticket - https://tracker.ceph.com/issues/37789
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

### CODE REVIEW

**Comment by votdev** [comment]
Should it look like this?
auswahl_005

**Comment by p-na** [comment]
> Should it look like this?
> auswahl_005

I would prefer the proposed look over what we currently have, but that might be a different PR.

File: src/pybind/mgr/dashboard/frontend/src/app/core/navigation/administration/administration.component.html [hunk 1]
```diff
@@ -6,6 +6,8 @@
      i18n-title
      title="Dashboard Settings">
     <i class="fa fa-cog"></i>
```
Comment (tspmelo) [inline src/pybind/mgr/dashboard/frontend/src/app/core/navigation/administration/administration.component.html]:
```suggestion
     <i class="fa fa-fw fa-cog"></i>
```

File: src/pybind/mgr/dashboard/frontend/src/app/core/navigation/dashboard-help/dashboard-help.component.html [hunk 2]
```diff
@@ -9,6 +9,8 @@
      i18n-title
      title="Help">
     <i class="fa fa-question-circle-o"></i>
```
Comment (tspmelo) [inline src/pybind/mgr/dashboard/frontend/src/app/core/navigation/dashboard-help/dashboard-help.component.html]:
```suggestion
     <i class="fa fa-fw fa-question-circle-o"></i>
```

**Comment by Exotelis** [comment]
> Could you add `fa-fw` to all the icons?
> then we would get an aligned list:
> image

I was looking for something like this. Thanks for the hint, will change it :-)

**Pull request merged**